fabric-item-api-v1/src/testmod/resources/fabric-item-api-tests-v1.mixins.json

```
@@ -0,0 +1,11 @@
+{
+  "required": true,
+  "package": "net.fabricmc.fabric.test.item.mixin",
+  "compatibilityLevel": "JAVA_16",
```

We actually migrated to Java 17 mixins before they did that
```suggestion
  "compatibilityLevel": "JAVA_17",
```